docs/network.md

```diff
@@ -0,0 +1,16 @@
+# Network ports
+
+Percona Platform requires configuration of network ports to accept monitoring data from PMM. 
+
+Make sure to configure your network and firewall so that all ports listed below are whitelisted and opened for bi-directional communication. UDP is not needed.
+
+Ports to expose:
+
+| PMM component | TCP port      | Direction     | Description
+|---------------|---------------|---------------|-----------------------------------------------------------------------------------------------------------------
+| PMM Server    |   3.135.83.118          | both          | HTTP server, used for gRPC over HTTP and web interface (**insecure**, use with caution).
```

Comment:
I do not think a direction column is needed as we already mention that communication should be bi-directional.